packages/eslint-plugin-office-addins/src/rules/no-empty-load.ts

```diff
@@ -0,0 +1,90 @@
+import { TSESTree } from "@typescript-eslint/experimental-utils";
+import {
+  Reference,
+  Scope,
+  Variable,
+} from "@typescript-eslint/experimental-utils/dist/ts-eslint-scope";
+import { isGetFunction, isLoadFunction } from "../utils";
+
+export = {
+  name: "no-empty-load",
+  meta: {
+    type: <"problem" | "suggestion" | "layout">"problem",
+    messages: {
+      emptyLoad: "Calling load without any argument can slow down your add-in",
+    },
+    docs: {
+      description:
+        "Calling load without any argument can cause load unneeded data and slow down your add-in",
+      category: <
+        "Best Practices" | "Stylistic Issues" | "Variables" | "Possible Errors"
+      >"Best Practices",
+      recommended: <false | "error" | "warn">false,
+      url: "https://docs.microsoft.com/en-us/office/dev/add-ins/develop/application-specific-api-model?view=powerpoint-js-1.1#calling-load-without-parameters-not-recommended",
+    },
+    schema: [],
+  },
+  create: function (context: any) {
+    function isEmptyLoad(node: TSESTree.MemberExpression): boolean {
+      if (
+        isLoadFunction(node) &&
+        node.parent?.type == TSESTree.AST_NODE_TYPES.CallExpression &&
+        node.parent.arguments.length === 0
+      ) {
+        return true;
+      }
+      return false;
+    }
+
+    function findEmptyLoad(scope: Scope) {
+      scope.variables.forEach((variable: Variable) => {
+        let getFound: boolean = false;
+        variable.references.forEach((reference: Reference) => {
+          const node: TSESTree.Node = reference.identifier;
+
+          if (
```

Comment:
If you look at what I figured out for call-sync-before-read you could use the isWrite() method and writeExpr property of the reference to combine these two set's of if blocks into one simpler one.  The outer if would check for isWrite() and the inner block would generically check isGetFunction with writeExpr as the argument.